docs/src/for-developers/variational_inference.md

```diff
@@ -20,257 +20,257 @@ In Bayesian inference one usually specifies a model as follows: given data ``$\{
 \end{align*}
 ```
 
-where ``$\overset{\text{i.i.d.}}{\sim}$`` denotes that the samples are identically independently distributed. Our goal in Bayesian inference is then to find the _posterior_
+where ``\overset{\text{i.i.d.}}{\sim}`` denotes that the samples are identically independently distributed. Our goal in Bayesian inference is then to find the _posterior_
 
 ```math
-p(z \mid \{ x_i \}_{i = 1}^n) = \prod_{i=1}^{n} p(z \mid x_i).
+p(z \mid \\{ x\_i \\}\_{i = 1}^n) = \prod\_{i=1}^{n} p(z \mid x\_i).
```

Comment:
Similarly, why is `\_` (and `\\`) used here but other formulas use `_` and `\`?

Reply:
The underscore has to be escaped if there are multiple ones in the same line, since when rendering the pages those will be interpreted as italic font first, which then breaks the rendering of the equation.